cryptography/fuzz/fuzz_targets/bls12381_primitive_operations.rs

```diff
@@ -226,7 +215,7 @@ enum FuzzOperation {
 
 impl<'a> Arbitrary<'a> for FuzzOperation {
     fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self, arbitrary::Error> {
-        let choice = u.int_in_range(0..=47)?;
+        let choice = u.int_in_range(0..=44)?;
 
         match choice {
```

Comment:
At some point, I hope there is a better way to do this match.

Reply:
(the fuzz testing could be cleaned up substantially, especially with #2377 )